src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/details/main.js

```diff
@@ -18,7 +18,8 @@ define([
         permissionData,
         matrixContainerSelector = '#matrix-container',
         matrixSelector = '#matrix',
-        formSelector = '#role-form';
+        formSelector = '#role-form',
+        setSelectAll = false;
```

Comment:
Why `set`? Doesn't make any sense to me. How about `allSelected` or `noneSelected`, depending on what you want to do.